<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt1Class</class>
 <widget class="QMainWindow" name="Qt1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnShow">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>点此计算</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>预算是多少？</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="budgetlineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>元</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>围栏形状（圆形/正方形/三角形）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outshapelineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>泳池参数（半径/边长）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inparalineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>泳池档次（简易/普通/高端）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="poorlevellineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>556</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>草地档次（简易/普通/高端）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="grasslevellineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>围栏参数（半径/边长）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outparalineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>泳池形状（圆形/正方形/三角形）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inshapelineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>646</x>
      <y>12</y>
      <width>581</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>单价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>围栏7元/米</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>简易泳池20元/平方米，普通泳池30元/平方米，高端泳池40元/平方米</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>简易草坪5元/平方米，普通草坪6元/平方米，高端草坪7元/平方米</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt1.qrc"/>
 </resources>
 <connections/>
</ui>
